CMakeLists.txt

```diff
@@ -321,8 +321,11 @@ if(webgui)
 endif()
 
 foreach(dir_to_copy ${directories_to_copy})
-  # For target dependencies:
+  # Create a list of files for which we want dependency tracking.
+  # Exclude certain endings, so CMake doesn't complain when they are deleted (JIRA#9688)
   file(GLOB_RECURSE artifacts_in ${dir_to_copy}/*)
+  list(FILTER artifacts_in EXCLUDE REGEX "\.(d$|swp|so|dylib|bak|old)$")
```

Comment:
Thanks a lot! Maybe add things like CMakeLists.txt.user or others listed in the .gitignore of ROOT, that generated when you open them in an IDE.
As well as .pcm files in case dictionaries are generated?

Reply:
I looked in .gitignore, and `xxx.user` doesn't seem to be listed. pcm is a good idea, I will add that!

Reply:
I had a second look, and I don't think that `.pcm` needs to be added. We are not suppressing it in `.gitignore`, and there's not a single pcm in the source tree.

Reply:
> and `xxx.user` doesn't seem to be listed

I see here this specific one: https://github.com/root-project/root/blob/master/.gitignore#L22

Reply:
I will set an arbitrary cutoff now 🙂: Not enough root developers use QTCreator to warrant the inclusion of this pattern.